### ISSUE TITLE
Changes for v1.1

### ISSUE THREAD

**Issue by SmittyHalibut** [open-headset-interconnect-standard/radio-adapter#2]
Follow-up from #1 these are the changes I want to make.

* Add component values to silkscreen, easier to assemble.
* Add through-holes to the Base board to Radio board interface. Castelated isn't working great on its own.  Adding through-holes that line up to allow someone to put a wire or pin through would be better.
* Add outline on Base board for the Shim board, partially to help the user place it, but also to make sure we designers remember not to put anything there.  Maybe even a keep-out zone.
* Swap M3x10mm with M3x12mm.  Adding the board thickness to the stack up leaves very few threads for the stand-off to grab.
  * The hardware here is the same as used for the User Adapter.  The 12mm will work in the User adapter too, even though it's not needed.  The advantage is that we can make a single pile of M3 hardware packets and use them for both the User and Radio adapter kits.

* GX16 connector riser boards are taller than the brass standoffs. They cause the top board to not lay down flat.  Push the GX16 out past the edge of the top board.  Or find some better way to address this.  (A slot in the top board to hold the riser board?)

* Does it make sense to revisit the over-all layout, do away with the "hand held" shape I developed for the user kits?  Maybe something more square?  Better suited to mounting next to a radio?  Maybe even with an optional metal case?

**Comment by ATLombardi** [comment]
I'm not sure how difficult this would be, but with how much clearance is underneath the GX16 it might be possible to lower it down and get away with using the existing standoffs. That might make connecting it to the radio board a little trickier though, since going down much further would put GX16 pin 4 very close to the radio board. Since the connection from pin 4 to the vertical slot is straight down anyway, that might not be a big issue...

**Comment by ATLombardi** [comment]
As for the overall layout, I think it'd be interesting to do a version with a female GX16 socket and design the whole adapter to sit almost flush with the radio itself, similar to the PowerPole conversion blocks I've seen for the Yaesu FT-891 and other radios. I know the obvious issue here is that every radio has their mic port in a different place, but fewer cables means a neater setup and fewer things to forget when going portable or helping with an event.

**Comment by SmittyHalibut** [comment]
> I'm not sure how difficult this would be, but with how much clearance is underneath the GX16 it might be possible to lower it down and get away with using the existing standoffs. That might make connecting it to the radio board a little trickier though, since going down much further would put GX16 pin 4 very close to the radio board. Since the connection from pin 4 to the vertical slot is straight down anyway, that might not be a big issue...

The vertical space is the concern. I lowered the GX16 as much as was convenient at the time not thinking about this problem. I'll spend some time lowering it as much as possible instead of as convenient. See what I can do.

**Comment by SmittyHalibut** [comment]
> As for the overall layout, I think it'd be interesting to do a version with a female GX16 socket and design the whole adapter to sit almost flush with the radio itself, similar to the PowerPole conversion blocks I've seen for the Yaesu FT-891 and other radios. I know the obvious issue here is that every radio has their mic port in a different place, but fewer cables means a neater setup and fewer things to forget when going portable or helping with an event.

That is an interesting idea. I like it. My immediate questions are: I don't think PCB mount GX16s are common, or possibly available at all. I don't REMEMBER seeing them, but I also wasn't looking for them so maybe they are. Have you looked?

I'd also worry about the headphone/speaker connection. Physical spacing between the GX16 and 3.5mm won't be consistent across radios, so the 3.5mm will still need to be a cable for radios that don't provide audio out through the mic port. 

But if we can find PCB mount female GX16, that would work pretty well for Icom radios (which do provide receive audio on the GX16.)

**Comment by gx1400** [comment]
**Icom GX16 Radio Adapter notes**
Tested with an Icom IC7300 with a Heil Pro Set Elite-Ic (electret) headset.

- No issues with PTT

- Audio is brought in through the GX16, but without a cable plugged into the PHONES port on the front of an ICOM, the IC7300 speaker will not disable. 

- Audio coming in through the 3.5mm audio jack is attenuated relative to the audio from the GX16 port (with no audio cable plugged into the GX16 adapter board. 
    - I think the R6/R8 resistors are only needed if using GX16 audio? The IC7300 PHONE board has the 100ohm resistors inside the IC7300. 
    
![image](https://github.com/user-attachments/assets/f2198ccc-e709-4427-b71a-8f71ae022188)

I'm confused a bit by the layout. Some install notes could be handy. This layout for the radio base board might need to be reviewed.

![image](https://github.com/user-attachments/assets/314cb91e-2b4c-4721-8ae2-b162a430f795)


- Standoff notes
    - Radio adapter PCB screws with the hex standoff spacing.  I replaced the M3x10 mm screw with an M3x12 mm screw and added an M3 nut to adjust the offset a little bit.  Suggest some PCB spacers.
    - The spacers are overall too short. I used 20mm long M3 hex spacers. 18mm would probably work well.

![PXL_20241217_021403236](https://github.com/user-attachments/assets/5dac8a2a-69ae-4297-8f16-b05160759d3b)
![PXL_20241217_021133129](https://github.com/user-attachments/assets/4d9e01c6-5955-44c4-a492-9d4ebb36d029)
![PXL_20241217_024648123](https://github.com/user-attachments/assets/cc721272-ef47-4d6b-8d29-2f60ff98c32f)

- I did not think to remove the nut from the connector.  Suggest notes to do so.
- Schematic shows red X over R7 and R5, but the PCB has those two ref des on the silk screen.  Suggest moving the rest X or change the ref des
![image](https://github.com/user-attachments/assets/2b3d4dd1-b99e-4093-8cdd-9927ae2d4635)

**Comment by gx1400** [comment]
> **Icom GX16 Radio Adapter notes** Tested with an Icom IC7300 with a Heil Pro Set Elite-Ic (electret) headset.
> 
>     * No issues with PTT
> 
>     * Audio is brought in through the GX16, but without a cable plugged into the PHONES port on the front of an ICOM, the IC7300 speaker will not disable.
> 
>     * Audio coming in through the 3.5mm audio jack is attenuated relative to the audio from the GX16 port (with no audio cable plugged into the GX16 adapter board.
>       
>       * I think the R6/R8 resistors are only needed if using GX16 audio? The IC7300 PHONE board has the 100ohm resistors inside the IC7300.
> 
> 
> ![image](https://private-user-images.githubusercontent.com/25909885/396331225-f2198ccc-e709-4427-b71a-8f71ae022188.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0MDQ0NzcsIm5iZiI6MTczNDQwNDE3NywicGF0aCI6Ii8yNTkwOTg4NS8zOTYzMzEyMjUtZjIxOThjY2MtZTcwOS00NDI3LWI3MWEtOGY3MWFlMDIyMTg4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE3VDAyNTYxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZmMGQ2OWQ0OTJhOTQzOWYwNmM2Y2JhOTRjYTFlMTI1YThjNWY2YmM4ZDI5NGZiNGJmMzQ5YTgxMTVlN2ViNzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.1EUhjYCHAOnD9_m_DlLws6Rc_LPuAFW2pt4AVE9Dqro)
> 
> I'm confused a bit by the layout. Some install notes could be handy. This layout for the radio base board might need to be reviewed.
> 
> ![image](https://private-user-images.githubusercontent.com/25909885/396333456-314cb91e-2b4c-4721-8ae2-b162a430f795.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0MDQ0NzcsIm5iZiI6MTczNDQwNDE3NywicGF0aCI6Ii8yNTkwOTg4NS8zOTYzMzM0NTYtMzE0Y2I5MWUtMmI0Yy00NzIxLThhZTItYjE2MmE0MzBmNzk1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE3VDAyNTYxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM5ODBjOTAyMmQ3NTMwZjJhYjA3ZGY4NTFmMzhkNWY1ZDkzYWI0YmQxNmQ1NTNlODdmNWY2MWIxYjQ2OTQ2Y2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.-gpOrab6EnZqa79pErAnHntg4gwK_23IedPEErv3OaY)
> 
>     * Standoff notes
>       
>       * Radio adapter PCB screws with the hex standoff spacing.  I replaced the M3x10 mm screw with an M3x12 mm screw and added an M3 nut to adjust the offset a little bit.  Suggest some PCB spacers.
>       * The spacers are overall too short. I used 20mm long M3 hex spacers. 18mm would probably work well.
> 
> 
> ![PXL_20241217_021403236](https://private-user-images.githubusercontent.com/25909885/396331782-5dac8a2a-69ae-4297-8f16-b05160759d3b.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0MDQ0NzcsIm5iZiI6MTczNDQwNDE3NywicGF0aCI6Ii8yNTkwOTg4NS8zOTYzMzE3ODItNWRhYzhhMmEtNjlhZS00Mjk3LThmMTYtYjA1MTYwNzU5ZDNiLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE3VDAyNTYxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFiN2NjYTNlMDA0ODkwM2E1NTRlMTYyZmU5M2MzODFhYmQyM2YwZGVmM2NmOGZlNTZkY2MxNDdhM2ViMGJmYTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.DI8VBwPS8zRqJfOIaxuw3uhFhnItAXiW4PuZPqN9o_o) ![PXL_20241217_021133129](https://private-user-images.githubusercontent.com/25909885/396331810-4d9e01c6-5955-44c4-a492-9d4ebb36d029.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0MDQ0NzcsIm5iZiI6MTczNDQwNDE3NywicGF0aCI6Ii8yNTkwOTg4NS8zOTYzMzE4MTAtNGQ5ZTAxYzYtNTk1NS00NGM0LWE0OTItOWQ0ZWJiMzZkMDI5LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE3VDAyNTYxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM3MDE2YThhZGExOTBiZjM3NjVkMmRhNTA3Y2YyMTI4ZDgxZGIxYjJjNzY5ODcwNjFmMzc0MWIzMmJmZjdkMmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GH-opr3nE5slXREOB_phue3wuj89I2Bh0dnn4toI0Cc) ![PXL_20241217_024648123](https://private-user-images.githubusercontent.com/25909885/396332172-cc721272-ef47-4d6b-8d29-2f60ff98c32f.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0MDQ0NzcsIm5iZiI6MTczNDQwNDE3NywicGF0aCI6Ii8yNTkwOTg4NS8zOTYzMzIxNzItY2M3MjEyNzItZWY0Ny00ZDZiLThkMjktMmY2MGZmOThjMzJmLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE3VDAyNTYxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg0ZWQxNTIyYWRhYTI1MzcwMGExNjJmZGQyZTU0YzIzZTZhYjYzZjlhYzAwMzE3OGZiNTgxYjhlNzU5ZjZhYTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.6nMdTuaJMBn-NMzo3kX0Y5WsgcDF_fwIqJ3b7NY-Nr4)
> 
>     * I did not think to remove the nut from the connector.  Suggest notes to do so.
> 
>     * Schematic shows red X over R7 and R5, but the PCB has those two ref des on the silk screen.  Suggest moving the rest X or change the ref des
>       ![image](https://private-user-images.githubusercontent.com/25909885/396332523-2b3d4dd1-b99e-4093-8cdd-9927ae2d4635.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0MDQ0NzcsIm5iZiI6MTczNDQwNDE3NywicGF0aCI6Ii8yNTkwOTg4NS8zOTYzMzI1MjMtMmIzZDRkZDEtYjk5ZS00MDkzLThjZGQtOTkyN2FlMmQ0NjM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE3VDAyNTYxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNhNDgwYWQ2OGZlOWVmYjE2NjViZmU3NmEwZWU4YjI0YzU1NDM3NWZmM2FmZTNhNzY2ZDljMWRjMTllMTEwODAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.egFFU2nr2yB9rjKh_kxhzWqHTmoNfrMEwE312c1XH4E)

Maybe an Icom specific configured board should have some of the following notes:

- Do not install: C1, C2, RV2, R4
- Install jumper wire on J3 from pin 2 to pin 4
- Unsure about R7 & R5 on headphones

**Comment by SmittyHalibut** [comment]
If I don't respond to an issue, it's because I agree with it and felt it didn't need any further discussion.

> Audio is brought in through the GX16, but without a cable plugged into the PHONES port on the front of an ICOM, the IC7300 speaker will not disable.

This is a known issue.  I'm not sure whether the built-in speaker can be disabled in software while still allowing audio out the GX16.  It might require a dummy-plug in the headphones port or something.

> Audio coming in through the 3.5mm audio jack is attenuated relative to the audio from the GX16 port (with no audio cable plugged into the GX16 adapter board. I think the R6/R8 resistors are only needed if using GX16 audio? The IC7300 PHONE board has the 100ohm resistors inside the IC7300.

The 100R resistors are bypassable.  That's what the jumpers on J6, pins 1-2, 3-4.  They short across the 100R resistors.  You need the 100R if you're feeding from a speaker output, and don't need them if feeding from a headphone output.  So putting jumpers on that side of the header shorts out the 100R series resistors.

Does that change the audio levels for you?

> I'm confused a bit by the layout. Some install notes could be handy. This layout for the radio base board might need to be reviewed.

And how...  Yes, this needs assembly instructions, which I haven't written yet.  I probably should have thrown together some notes before sending the hardware to y'all.

I'm all for changes to the board layout if it'll make it easier to understand what's going on.  Any specific suggestions? (other than what you've already presented here, that is.)

> Radio adapter PCB screws with the hex standoff spacing. I replaced the M3x10 mm screw with an M3x12 mm screw and added an M3 nut to adjust the offset a little bit. Suggest some PCB spacers.

M3x12mm: I came to that conclusion too.  I've updated the BOM.

Spacer: Already done, and almost ready to ship to me.
![image](https://github.com/user-attachments/assets/60df5b04-bdd5-45bd-bb56-a667b9458f8d)

These are the same thickness as the radio-specific board and go on the other stand-offs to even them out.  Yes, I made sure the dog-bone would fit. :-)

> The spacers are overall too short. I used 20mm long M3 hex spacers. 18mm would probably work well.

Yeah, the stand-offs were sized for RJ45s; I didn't think about the GX16 being taller.

> Schematic shows red X over R7 and R5, but the PCB has those two ref des on the silk screen. Suggest moving the rest X or change the ref des

Good catch. That was the intent from the beginning, I just missed these.  Thanks.  Fixed.

> Maybe an Icom specific configured board should have some of the following notes:
>
>    Do not install: C1, C2, RV2, R4
>    Install jumper wire on J3 from pin 2 to pin 4

That's exactly what the jumper header does.  Note the silkscreen on the board shows you the orientation to install a jumper to bypass the mic attenuation circuit.

![image](https://github.com/user-attachments/assets/ef6b0f3e-ff25-4880-87cc-36508a354614)

Your jumpers are installed correctly (horizontally).  Does adjusting RV1 make a difference to your mic audio level?  Does removing those components change anything?

Or, are you just saying because they will NEVER be needed, don't bother populating them and just hard jumper the board instead?

That's actually a compelling argument:  Since the radio-specific board is soldered in place, any given kit will only ever work in one radio configuration.  Having it be reconfigurable doesn't make as much sense as it does on the User Adapter.

**Comment by SmittyHalibut** [comment]
Bah. @gx1400 ^^  Forgot to mention you. Sorry.

**Comment by gx1400** [comment]
> > Audio is brought in through the GX16, but without a cable plugged into the PHONES port on the front of an ICOM, the IC7300 speaker will not disable.
> 
> This is a known issue. I'm not sure whether the built-in speaker can be disabled in software while still allowing audio out the GX16. It might require a dummy-plug in the headphones port or something.

I feel like maybe those 100ohm resistors should live on the GX16 add-on board, then get disconnected (to use the ones that are inside of the IC7300) when an audio cable is inserted.

If not, I think for the ICOM IC7300, I'd recommend cutting the trace (or putting a ohm jumper pad there) so that it needs to the headphone audio cable for audio.

> > Audio coming in through the 3.5mm audio jack is attenuated relative to the audio from the GX16 port (with no audio cable plugged into the GX16 adapter board. I think the R6/R8 resistors are only needed if using GX16 audio? The IC7300 PHONE board has the 100ohm resistors inside the IC7300.
> 
> The 100R resistors are bypassable. That's what the jumpers on J6, pins 1-2, 3-4. They short across the 100R resistors. You need the 100R if you're feeding from a speaker output, and don't need them if feeding from a headphone output. So putting jumpers on that side of the header shorts out the 100R series resistors.
> 
> Does that change the audio levels for you?

I finally figured that out last night.  It's just some clever layout that's hard to follow.

It does change the audio level, But maybe it's the 100ohm resistors in the IC7300 vs the lack thereof over the GX16 audio path that's throwing me for a loop.  


> Spacer: Already done, and almost ready to ship to me. ![image](https://private-user-images.githubusercontent.com/497398/396599900-60df5b04-bdd5-45bd-bb56-a667b9458f8d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0NTcyNDcsIm5iZiI6MTczNDQ1Njk0NywicGF0aCI6Ii80OTczOTgvMzk2NTk5OTAwLTYwZGY1YjA0LWJkZDUtNDViZC1iYjU2LWE2NjdiOTQ1OGY4ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxN1QxNzM1NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zOGNkM2RlN2MxMmNlYjFjMmNlMjY2NjA5M2ViOTNjYTkwNTk2MDZlNjkzMTA5MWRkOWI0YTk4ZmRmM2Q4NzVjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.n-PaJaUapNLP9l6AATuMf9NhBNGf6QG7u6CSjiGlDlw)
> 
> These are the same thickness as the radio-specific board and go on the other stand-offs to even them out. Yes, I made sure the dog-bone would fit. :-)

Looks nice!

> > Maybe an Icom specific configured board should have some of the following notes:
> > Do not install: C1, C2, RV2, R4
> > Install jumper wire on J3 from pin 2 to pin 4
> 
> That's exactly what the jumper header does. Note the silkscreen on the board shows you the orientation to install a jumper to bypass the mic attenuation circuit.
> 
> ![image](https://private-user-images.githubusercontent.com/497398/396602954-ef6b0f3e-ff25-4880-87cc-36508a354614.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0NTcyNDcsIm5iZiI6MTczNDQ1Njk0NywicGF0aCI6Ii80OTczOTgvMzk2NjAyOTU0LWVmNmIwZjNlLWZmMjUtNDg4MC04N2NjLTM2NTA4YTM1NDYxNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxN1QxNzM1NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05Nzk1MTg0NmQ1MjUzMDIxYjQzYThjZjI5ZjJkMTM3Yjg0YjExNTdkMTJmODRlODdjYjBlMWM5YzAwNDI3NDVlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.sBL7MGUW6H9618DOOsOhl-xP3j-kQK9rxg3_I9cDNdA)
> 
> Your jumpers are installed correctly (horizontally). Does adjusting RV1 make a difference to your mic audio level? Does removing those components change anything?
> 
> Or, are you just saying because they will NEVER be needed, don't bother populating them and just hard jumper the board instead?
> 
> That's actually a compelling argument: Since the radio-specific board is soldered in place, any given kit will only ever work in one radio configuration. Having it be reconfigurable doesn't make as much sense as it does on the User Adapter.

I think RV1 circuit is totally by-passed in the Electret configuration?  I think your last statement is the takeaway.  If the boards are soldered for specific radio configurations, maybe all of the user configurable jumpers would add confusion.  

I think the README for any specific radio or family of radios could tell you (and show you) which jumpers and components to install where.

Removing the right angle connectors and pin jumpers might help bring down bom cost a bit too

**Comment by SmittyHalibut** [comment]
> I feel like maybe those 100ohm resistors should live on the GX16 add-on board, then get disconnected (to use the ones that are inside of the IC7300) when an audio cable is inserted.
> 
> If not, I think for the ICOM IC7300, I'd recommend cutting the trace (or putting a ohm jumper pad there) so that it needs to the headphone audio cable for audio.

I remembered now.  I'm not sure that all Icom radios put out audio on GX16.  I know its common with modern gear, but I don't know how universal that has been over the years.  Similarly, I'm pretty sure that not all radios have Headphones output, some might only have Speaker.  The 100R are required there.

So the question isn't: What's needed to support THIS PARTICULAR ICOM, but what's needed to support ALL Icoms?  I don't think it's a good idea to limit the options.

> I finally figured that out last night. It's just some clever layout that's hard to follow.

Yeah, it took me a bit to figure it out again today.  I could draw that better, that's for sure.

> It does change the audio level, But maybe it's the 100ohm resistors in the IC7300 vs the lack thereof over the GX16 audio path that's throwing me for a loop.

> > Your jumpers are installed correctly (horizontally). Does adjusting RV1 make a difference to your mic audio level? Does removing those components change anything?
> > Or, are you just saying because they will NEVER be needed, don't bother populating them and just hard jumper the board instead?
> > That's actually a compelling argument: Since the radio-specific board is soldered in place, any given kit will only ever work in one radio configuration. Having it be reconfigurable doesn't make as much sense as it does on the User Adapter.
> 
> I think RV1 circuit is totally by-passed in the Electret configuration? I think your last statement is the takeaway. If the boards are soldered for specific radio configurations, maybe all of the user configurable jumpers would add confusion.

I think the right answer here is to keep the header outline on the board, but document how to assemble it hard wired for a given config so the jumpers aren't needed.  And say things like: "If you configure for electret, you're bypassing this whole circuit so you can just leave those parts off the board if you want."

But I do want to keep the configuration options available.

> I think the README for any specific radio or family of radios could tell you (and show you) which jumpers and components to install where.

Agreed.  The documentation on these prototypes I sent you was ... well, "lacking" is too kind a word.  "Entirely non-existent" is more accurate.

> Removing the right angle connectors and pin jumpers might help bring down bom cost a bit too

They are a tiny fraction of the BOM cost.  I'd keep them in the kit, but document how to not use them if you don't want to.

**Comment by gx1400** [comment]
For Yaesu RJ45 kits - the kits you shipped have one RJ45 with LED leads, the other without. If plan it to sell kits with one of each, be sure to emphasize that the two RJ45s are different and that the LED should be noted and installed in the correct location
